### PR TITLE
Tamil [ta] Language Translations 

### PR DESCRIPTION
**Pull request by darthleslin**
**Description**
Tamil Language Translations.

**Checklist:**
- [x] Securely signed commits
- [x] Component(s) with PHPDoc blocks, only if necessary or adds value
- [x] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ claims no expertise beyond English.
 | sk     | Slovak              | 4.0.0   | [xbotkaj][sk]         | Jozef Botka
 | sr     | Serbian             | 4.3.1   | [nebojsajsimic][sr]   | Nebojša Simić
 | sv-SE  | Swedish - Sweden    | 4.0.3   | [tangix][sv-SE]       | Mattias Sandström
+| ta     | Tamil               | 4.5.0   | [darthleslin][ta]     | Leslin
```
Comment (paulbalandan) [inline README.md:69]:
```suggestion
| ta     | Tamil               | 4.5.1   | [darthleslin][ta]     | Leslin
```

File: README.md [hunk 2]
```diff
@@ -102,7 +103,7 @@ _**Version where the locale was released. 4.x denotes `Unreleased`._
 [si]: https://github.com/npwsamarasinghe
 [sk]: https://github.com/xbotkaj
 [sr]: https://github.com/nebojsajsimic
-[sv-SE]: https://github.com/tangix
+[ta]: https://github.com/darthleslin
```
Comment (paulbalandan) [inline README.md:106]: don't delete sv-SE, just insert your own

File: README.md (hunk 1 above)
Comment (paulbalandan) [reply README.md:69]: Should this be your full name?

**Review by paulbalandan — CHANGES_REQUESTED** [review]
@darthleslin thanks for this grrat addition! I have some minor comments before we can merge this.

File: README.md (hunk 2 above)
Comment (darthleslin) [reply README.md:107]: Changes DONE.
Thank You.

File: README.md (hunk 1 above)
Comment (darthleslin) [reply README.md]: Hello Paul.
Yes. Leslin is my full name.

**Review by paulbalandan — APPROVED** [review]

**Comment by paulbalandan** [comment]
Thank you, @darthleslin

**Pull request merged**